src/actions/promocode-actions.js

```diff
@@ -241,6 +241,8 @@ export const getPromocode = (promocodeId) => async (dispatch, getState) => {
   const params = {
     expand:
       "owner,sponsor,sponsor.company,sponsor.sponsorship,sponsor.sponsorship.type,speaker,tickets,ticket_type,ticket_types_rules,tags,allowed_ticket_types",
+    relations:
+      "owner,sponsor,speaker,tickets,ticket_types_rules,tags,allowed_ticket_types",
```

Comment:
@santipalenque 
i dont think that speaker nor tickets collections are used anymore 
could we get rid of these too ?